src/pretix/static/pretixcontrol/js/ui/main.js

```diff
@@ -355,7 +355,16 @@ var form_handlers = function (el) {
         var dependent = $(this),
             dependency = $($(this).attr("data-display-dependency")),
             update = function (ev) {
-                var enabled = dependency.toArray().some(function(d) {return (d.type === 'checkbox' || d.type === 'radio') ? d.checked : (!!d.value && !d.value.match(/^0\.?0*$/g))});
+                var enabled = dependency.toArray().some(function(d) {
+                    if (d.type === 'checkbox' || d.type === 'radio') {
+                        return d.checked;
+                    } else if (d.type === 'select-one') {
+                        return d.value === dependent.attr("data-display-dependency-value");
```

Comment:
I do not see where this is in use. Also an input-element of type `select-one` is not really common. Could you elaborate on this? Thanks! :)

Reply:
This is the code that makes the from/to fields show up when selecting "other".

`"select-one"` is the `DOMElement.type` value of a `<select>` without the `multiple` attribute. Wasn't my idea, and I also learned it yesterday: https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/type

Reply:
Wow, thanks, TIL!